dune

```diff
@@ -0,0 +1 @@
+(vendored_dirs rocq stdlib coq-lsp)
```

Comment:
`make all` gives the error:

```
dune build -p rocq-runtime,coq-core,rocq-core,coq,rocq-stdlib,coq-waterproof
Error: Package coq is vendored and so will never be masked. It is redundant
to pass it to --only-packages.
Error: Package rocq-stdlib is vendored and so will never be masked. It is
redundant to pass it to --only-packages.
```

which seems related to this

Reply:
Updating the build target in the Makefile to do `dune build` fixes this

Reply:
We have chosen to go for
```
dune build -p coq-waterproof
```